activestorage/test/models/attachments_test.rb

```diff
@@ -187,6 +187,14 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
     end
   end
 
+  test "selectively purge attached blob from has_many association" do
+    @user.highlights.attach create_blob(filename: "funky.jpg"), create_blob(filename: "wonky.jpg")
+
+    @user.highlights.purge(1)
+
+    assert_nothing_raised
```

Comment:
`assert_nothing_raised` requires a block like `assert_nothing_raised { @user.highlights.purge(1) }`.

Reply:
Thank you! Turned out to be a silly mistake, I was hard-coding the attachment id, but they are not cleared out until the end so it didn't match. Fixed now